mp/src/materialsystem/stdshaders/pbr_vs20.fxc

```diff
@@ -0,0 +1,137 @@
+//===================== Copyright (c) Valve Corporation. All Rights Reserved. ======================
+//
+// Physically Based Rendering shader for brushes and models
+//
+//==================================================================================================
+
+// static combos
+// STATIC: "LIGHTMAPPED"	            "0..1"
+
+// dynamic combos
+// DYNAMIC: "SKINNING"					"0..1"
+// DYNAMIC: "COMPRESSED_VERTS"			"0..1"
+// DYNAMIC: "DOWATERFOG"				"0..1"
+// DYNAMIC: "LIGHTING_PREVIEW"			"0..1"
+// DYNAMIC: "NUM_LIGHTS"				"0..4"
+
+// getting rid of the blue squiggles
+#ifndef LIGHTMAPPED
+#define LIGHTMAPPED 0
+#endif//LIGHTMAPPED
+#ifndef SKINNING
+#define SKINNING 0
+#endif//SKINNING
+#ifndef COMPRESSED_VERTS
+#define COMPRESSED_VERTS 0
+#endif//COMPRESSED_VERTS
+#ifndef DOWATERFOG
+#define DOWATERFOG 0
+#endif//DOWATERFOG
+#ifndef LIGHTING_PREVIEW
+#define LIGHTING_PREVIEW 0
+#endif//LIGHTING_PREVIEW
+#ifndef NUM_LIGHTS
+#define NUM_LIGHTS 0
+#endif//NUM_LIGHTS
+
+// includes
+#include "common_vs_fxc.h"
+
+// shader specific constants
+const float4 cBaseTexCoordTransform[2] : register(SHADER_SPECIFIC_CONST_0);
+
+// shader input
+struct VS_INPUT
+{
+	// this is all of the stuff that we ever use
```

Comment:
Inconsistent space/tab usage in this file too.

Reply:
right, I'll try to fix it (again) (visual studio hates me)